System.IO.FileSystem/nanoFramework/SDCardCDParameters.cs

```diff
@@ -0,0 +1,41 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace nanoFramework.System.IO.FileSystem
+{
+    /// <summary>
+    /// Parameter used for Card detection when creating a SDcard instance.
+    /// </summary>
+    public class SDCardCDParameters
```

Comment:
The name could become large, still its preferable than using initials to condense the action that is, in fact, the reason for this class to exist: Card Detection. I suggest `CardDetectionParameters`.

Reply:
Done